test/transforms.jl

```diff
@@ -951,12 +951,14 @@
 
   @testset "Scale" begin
     # constant column
-    x = fill(3.0, 10)
+    x = fill(Float64(3.0), 10)
     y = rand(10)
-    t = Table(; x, y)
+    t = Table(; x=x, y=y, xf32=Float32.(x), xf16=Float16.(x))
```

Comment:
We don't alter existing tests in PRs. Please write a separate test for the fix you are proposing.

Reply:
I added a separate test to check that the eltype of the column does not change, but still in the same testset. Is that what you wanted?